README.md

```diff
@@ -822,7 +822,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
 
 ## Search
 
-  * [algolia.com](https://www.algolia.com/) — Hosted search-as-you-type (instant). Free hacker plan up to 10,000 documents and 100,000 operations. Bigger free plans available for community/Open Source projects
+  * [algolia.com](https://www.algolia.com/) — Hosted, highly efficient search solution, with typo-tolerance and relevance. Provides UI libraries to easily create search experiences. Free "Build" plan includes 1M documents and 10K searches/month. Also offers [developer documentation search](https://docsearch.algolia.com/) for free.
```

Comment:
Thanks for the update, I think we can remove some of the redundant words that just make the entry longer.  Like I dont think `highly efficient` really has value, would any commercial offering not claim to be efficient?

Lets focus on features and not marketing speak.

Reply:
Makes sense! I guess I was a little too inspired from the other entries in the search section 😄: 
* "... lightning-fast, and hyper-relevant search engine ..." https://github.com/ripienaar/free-for-dev/commit/8d0bb6dea8569133b055368d0b8824ae4351bb04
* "... which significantly decreases the time users find anything in an app" https://github.com/ripienaar/free-for-dev/commit/8d0bb6dea8569133b055368d0b8824ae4351bb04

No matter, updated here: https://github.com/ripienaar/free-for-dev/pull/3070/commits/ab5df71cf95d74b8fa8c02f6d799773d8e9ee2f5

Reply:
I've grown ever more weary of marketing speak :) thanks for fixing